core/gather/driver/execution-context.js

```diff
@@ -16,6 +16,13 @@ class ExecutionContext {
 
     /** @type {number|undefined} */
     this._executionContextId = undefined;
+    /**
```

Comment:
what about just a counter incremented in `_getOrCreateIsolatedContextId` when created?